### ISSUE TITLE
lack of utils file or directory

### ISSUE THREAD

**Issue by xaddwell** [TrustAIoT/LRS#1]
![屏幕截图 2024-07-26 172825](https://github.com/user-attachments/assets/11ada32b-bd28-44c5-ac9e-7f81ee608029)

cannot import this file @mstwutao

**Comment by xaddwell** [comment]
tulip and cure are the key loss func in this paper

**Comment by xaddwell** [comment]
Towards Understanding and Boosting Adversarial Transferability from a Distribution Perspective (TIP 2022)

**Comment by lyygua** [comment]
> Towards Understanding and Boosting Adversarial Transferability from a Distribution Perspective (TIP 2022)

Hi xaddwell, I’m interested in the implementation details of the loss function. Have you managed to reproduce the results? If so, could you share any insights or tips on how to effectively reproduce the code? Thanks!

**Comment by xaddwell** [comment]
> Hi xaddwell, I’m interested in the implementation details of the loss function. Have you managed to reproduce the results? If so, could you share any insights or tips on how to effectively reproduce the code? Thanks!

Actually, this paper opensource the ckpt of the finetuned model, without corresponding code.

**Comment by lyygua** [comment]
Hi, have you succeeded in reproducing the result in the paper?

**Comment by xaddwell** [comment]
> Hi, have you succeeded in reproducing the result in the paper?

I have recently released the code of my paper (AAAI 2025) with reproducible experiments. We also fine-tuned the model to enhance transferability, but we pointed out that once fine-tuning was suboptimal, https://github.com/xaddwell/AWT.git
